treeherder/perf/models.py

```diff
@@ -26,6 +27,33 @@ def __str__(self):
         return self.name
 
 
+def has_one_month_worth_of_data(perf_data):
```

Comment:
These helper functions should be used as private/protected methods inside the main class.